FHIR 101 - Practical Guide.ipynb

```diff
@@ -1479,7 +1479,9 @@
     "hidden": true
    },
    "source": [
-    "At this point we're going to take a look at how search works in FHIR so that we can start searching for our resources using the attributes we're adding to our models. Also, this will come in handy for debugging - to find and verify that we've created resources we intended to. \n",
+    "At this point we're going to take a look at how search works in FHIR so that we can start searching for our resources using the attributes we're adding to our models. Also, this will come in handy for debugging - to find and verify that we've created resources we intended to.\n",
+    "\n",
+    "📘 The examples in this section will use a public FHIR server called [HAPI](http://hapi.fhir.org/). Please note that this public server is periodically purged and contains test data, so the search results may vary. If you have a local FHIR server, you can replace the HAPI URLs in the following examples with your local server URL.\n",
```

Comment:
I am not sure if we need this first sentence since we use HAPI across the entire tutorial. We may want to describe as follows:

"Please note that the HAPI server is periodically purged and contains test data, so the search results may vary. If you have a local FHIR server, you can replace the HAPI URLs in the following examples with your local server URL."